eZ/Publish/Core/Persistence/Cache/ContentLanguageHandler.php

```diff
@@ -103,9 +103,9 @@ function (array $ids) {
     public function loadByLanguageCode($languageCode)
     {
         return $this->getCacheValue(
-            $languageCode,
+            $this->escapeForCacheKey($languageCode),
             'ez-language-code-',
-            function ($languageCode) {
+            function () use ($languageCode) {
```

Comment:
Is this change necessary?

Reply:
without this change, we will use the variable after using the escapeForCacheKey function, and we need the original value to search in the database